akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -716,13 +716,21 @@ private[akka] object Running {
           this
         } else {
           visibleState = state
-          if (shouldSnapshotAfterPersist == NoSnapshot || state.state == null) {
+          def skipRetention(): Boolean = {
+            // only one retention process at a time
+            val inProgress = shouldSnapshotAfterPersist == SnapshotWithRetention && setup.isRetentionInProgress()
+            if (inProgress)
+              setup.internalLogger
+                .info("Skipping retention at seqNr [{}] because previous retention has not completed yet.", state.seqNr)
```

Comment:
```suggestion
                .info("Skipping retention at seqNr [{}] because previous retention has not completed yet. Next retention will cover skipped retention.", state.seqNr)
```
Or something like that, so it doesn't sound so much like a problem to act on?

Reply:
done 👍